.github/workflows/release.yml

```diff
@@ -30,18 +30,17 @@ jobs:
           sudo apt update
           sudo apt remove -y firefox
           sudo apt upgrade -y
-          sudo apt install -y \
+          sudo apt install -y --no-install-recommends \
```

Comment:
This addition won't make much difference here, but it doesn't seem to harm anything.

When building the docker image though, there's some nice savings:

## Before 
```
65 Summary:
7.165   Upgrading: 0, Installing: 346, Removing: 0, Not Upgrading: 0
7.165   Download size: 673 MB
7.165   Space needed: 3240 MB / 146 GB available
```

## After
```
 #6 6.604 Summary:
#6 6.604   Upgrading: 0, Installing: 279, Removing: 0, Not Upgrading: 0
#6 6.604   Download size: 563 MB
#6 6.604   Space needed: 2568 MB / 30.0 GB available
```
https://github.com/KaruroChori/vs-fltk/actions/runs/12506343689/job/34891143298#step:7:484